### DISCUSSION TITLE
How to parse the output of the command POLLUPDATES

### DISCUSSION THREAD

**Discussion by ninuxer** [apache/kvrocks#2684] [Q&A] [answered]
For the new POLLUPDATES added in 2.10, it is easy to get the incremental update content, but the parsed content is in encoded format. How to identify the updates content so that it can be easily combined with the cdc program?
![image](https://github.com/user-attachments/assets/28c7ce2c-e5f2-4f2f-9aff-8b33e29942d6)

**Comment by git-hulk** [comment, marked as answer]
@ninuxer it's now in rocksdb batch format, will support the command format at the next release or the release after the next.

**Comment by PragmaTwice** [comment]
It's now supported in the unstable branch (https://github.com/apache/kvrocks/commit/c22f5e14f67665e1d2a95a923fd44d37aee0eaf1) and is expected to be released in 2.12.0.
